script.js

```diff
@@ -0,0 +1,141 @@
+class ProductAPI {
+  constructor(baseURL) {
+    this.baseURL = baseURL;
+  }
+
+  async apiGet(limit, offset) {
+    try {
+      const response = await fetch(
+        `${this.baseURL}?limit=${limit}&skip=${offset}&select=title,price`
+      );
+      if (!response.ok) {
+        throw new Error(`Error fetching products: ${response.statusText}`);
+      }
+      const data = await response.json();
+      return data;
+    } catch (error) {
+      console.error("Error fetching products:", error);
+      throw error;
+    }
+  }
+
+  async getProduct(id) {
+    try {
+      const response = await fetch(`${this.baseURL}/${id}`);
+      if (!response.ok) {
+        throw new Error(`Error fetching product: ${response.statusText}`);
+      }
+      const data = await response.json();
+      return data;
+    } catch (error) {
+      console.error("Error fetching product:", error);
+      throw error;
+    }
+  }
+}
+
+const baseURL = "https://dummyjson.com/products";

```

Comment:
move every dependent variable in the ProductClass

Reply:
done